client/src/components/AddPollDialog.js

```diff
@@ -162,7 +170,13 @@ class AddPollDialog extends Component {
                         return;
                     }
 
-                    // add new poll to Profile and close dialog:
+                    // add the list title and voter counts to the returned data
+                    response.data.data.sendToList = {
```

Comment:
Can we do something to simplify `response.data.data`? Looks weird...

Reply:
Changed key of returned data to newPoll ---> response.data.newPoll